pages/plugins/using.md

```diff
@@ -111,8 +110,25 @@ steps:
   - label: "Read something else"
     command: echo "On to a new book"
     plugins:
-      - *docker 
+      - *docker
+```
+
+You can override an anchor by using `<<:` before the _alias_. This allows overriding parts of the anchor as required, reducing the need to create multiple anchors with similar configurations.
+
+The following example uses a YAML anchor (`docker-step`) and overrides the command run whilst using the same plugin version and container image:
 
+```yml
+common:
+  - docker-step: &docker-step
+      command: "uname -a"
+      plugins:
+        docker#v5.11.0:
+          image: alpine
+
+steps:
+  - *docker-step
+  - <<: *docker-step
+    command: "date"
```

Comment:
Actually, thinking about this PR a little more, it might be worthwhile explaining in a little more detail what this actually 'expands' out to (in a subsequent YAML block), to help back up your explanation...

I'm a little rusty on YAML syntax but would this bit of YAML code (from your definitions above) be:

```yml
steps:
  - command: "date"
    plugins:
      docker#v5.11.0:
        image: alpine
```

or

```yml
steps:
  - command: "date"
  - command: "uname -a"
    plugins:
      docker#v5.11.0:
        image: alpine
```

Please correct me if I'm wrong

Reply:
Actually, using the online YAML parser, I discovered this actually expands to:

  "steps": [
    {
      "command": "uname -a", 
      "plugins": {
        "docker#v5.11.0": {
          "image": "alpine"
        }
      }
    }, 
    {
      "command": "date", 
      "plugins": {
        "docker#v5.11.0": {
          "image": "alpine"
        }
      }
    }
  ], 

```yml
steps:
  - command: "uname -a"
    plugins:
      docker#v5.11.0:
        image: alpine
   - command: "date"
    plugins:
      docker#v5.11.0:
        image: alpine
```